storagenode/orders/db_test.go

```diff
@@ -78,8 +83,44 @@ func TestOrders(t *testing.T) {
 
 		unsent, err := ordersdb.ListUnsent(ctx, 100)
 		require.NoError(t, err)
-
 		require.Empty(t, cmp.Diff([]*orders.Info{info}, unsent, cmp.Comparer(pb.Equal)))
+
+		// list by group
+		unsentGrouped, err := ordersdb.ListUnsentBySatellite(ctx)
+		require.NoError(t, err)
+		require.Empty(t, cmp.Diff(map[storj.NodeID][]*orders.Info{
```

Comment:
it would be nice to format this or separate into different var because its super hard to read this